src/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Squad chat</title>
+    <style>
```

Comment:
My suggestion - make separate folder for all public files, like this html file, and move styling to own file, in case styling will expand and this index.html will become unreadable.